package.json

```diff
@@ -69,7 +69,7 @@
     "most": "^1.0.2",
     "prettier": "^1.2.2",
     "prop-types": "^15.6.1",
-    "react": "^16.3.1",
+    "react": "^17.0.1",
```

Comment:
It's likely not wise to bump this so that anything between `16.3.1` and `17.0.1` isn't allowed.

Reply:
@joealden  thanks for the review, the tests are passing. Why do you think it would be risky to bump to the current react version for development of this package?